### PR TITLE
Add information about interrupted conversion/dist-upgrade to stdout and motd

### PR DESCRIPTION
**Pull request by SandakovMM**

### CODE REVIEW

**Review by ukablan-wpc — APPROVED** [review]

File: pleskdistup/main.py [hunk 1]
```diff
@@ -321,6 +321,14 @@ def try_lock(lock_file: PathType) -> typing.Generator[bool, None, None]:
 def exit_signal_handler(signum, frame):
     # exit will trigger blocks finalization, so lockfile will be removed
     log.info(f"Received signal {signum}, going to exit...")
+    print(f"The dist-upgrade process was stopped by signal {signum}. Please use the `--revert` option before trying again.")
+
+    motd.add_finish_ssh_login_message("""
+The dist-upgrade process was stopped by signal.
+Please use the `--revert` option before trying again.
+""")
```
Comment (ukablan-wpc) [inline pleskdistup/main.py:329]: What about mentioning about the program name?

**Review by ukablan-wpc — APPROVED** [review]

**Comment by ukablan-wpc** [comment]
Can you also share the console outputs when you try it?

**Comment by SandakovMM** [comment]
Sure, no problem:
```
# ./centos2alma 
Doing preparation checks...
^CThe dist-upgrade process was stopped by signal 2. Please use the `centos2alma --revert` option before trying again.
# exit
logout
Connection to plesk-cos-7-1 closed.
~ [1]$ ssh root@XXX

===============================================================================
Message from the Plesk dist-upgrader tool:

    The dist-upgrade process was stopped by signal.
    Please use the `centos2alma --revert` option before trying again.
    You can remove this message from the /etc/motd file.
===============================================================================
```

**Comment by ukablan-wpc** [comment]
> Sure, no problem:
> 
> ```
> # ./centos2alma 
> Doing preparation checks...
> ^CThe dist-upgrade process was stopped by signal 2. Please use the `centos2alma --revert` option before trying again.
> # exit
> logout
> Connection to plesk-cos-7-1 closed.
> ~ [1]$ ssh root@XXX
> 
> ===============================================================================
> Message from the Plesk dist-upgrader tool:
> 
>     The dist-upgrade process was stopped by signal.
>     Please use the `centos2alma --revert` option before trying again.
>     You can remove this message from the /etc/motd file.
> ===============================================================================
> ```

Does it revert motd message when we revert?

**Comment by SandakovMM** [comment]
Yes. `--revert` restores motd file from backup

**Pull request merged**